src/plugins/intel_cpu/src/extension.cpp

```diff
@@ -114,6 +114,10 @@ std::map<std::string, ngraph::OpSet> Extension::getOpSets() {
         NGRAPH_OP(MVN, ngraph::op::v6)
         NGRAPH_OP(Select, ngraph::op::v1)
         NGRAPH_OP(ConvolutionBackpropData, ngraph::op::v1)
+        NGRAPH_OP(LSTMSequence, ngraph::op::v5)
+        NGRAPH_OP(GRUSequence, ngraph::op::v5)
+        NGRAPH_OP(NonMaxSuppression, ngraph::op::v5)
+        NGRAPH_OP(NonMaxSuppression, ngraph::op::v9)
```

Comment:
Besides LPT, ConvertPrecision pass often wrap operations as TypeRelaxed (e.g. [NMS](https://github.com/openvinotoolkit/openvino/blob/master/src/common/transformations/src/transformations/convert_precision.cpp#L638)). Sequences are touched by [ConvertFqRnnToQuantizedRnn](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/transformations/cpu_opset/common/pass/convert_fq_rnn_to_quantized_rnn.cpp#L110) transformation

Reply:
@dmitry-gorokhov Taking this opportunity, I also want to remind that we cannot cover this code with tests at the moment, because when an exception "Op from unsupported opset: type_relaxed" is thrown at the stage of importing a model from the cache, it is caught and the cache is recollected again.
As a solution, it was proposed to catch only "blob outdated" exceptions and throw other exceptions. It was described in CVS-54335. Maybe we should prioritize this ticket?